site/docs/get-started/quickstart.md

```diff
@@ -0,0 +1,70 @@
+# Quick Start Guide
+
+Follow this guide to install Mesh for Data using default parameters that are suitable for experimentation.
+<!-- For a full installation refer to the [full installation guide](./setup/install) instead. -->
+
+## Before you begin
+
+Ensure that you have the following:
+
+- [Helm](https://helm.sh/) 3.3 or newer must be installed and configured on your machine.
+- [Kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) 1.16 or newer must be installed on your machine.
+- Access to a Kubernetes cluster such as [Kind](http://kind.sigs.k8s.io/) as a cluster administrator.
+
+## Install cert-manager
+
+Mesh for Data requires [cert-manager](https://cert-manager.io) to be installed to your cluster. 
+Many clusters already include cert-manager. Run the following to install cert-manager only if it's missing:
+
+```bash
+kubectl get namespace cert-manager || kubectl apply -f https://github.com/jetstack/cert-manager/releases/download/v1.2.0/cert-manager.yaml
+``` 
+
+## Install Hashicorp Vault and plugins
+
+[Hashicorp Vault](https://www.vaultproject.io/) and a [secrets-kubernetes-reader](https://github.com/mesh-for-data/vault-plugin-secrets-kubernetes-reader) plugin are used by Mesh for Data for credential management.
+
+Run the following to install vault and the plugin in development mode:
+
+=== "Kubernetes" 
+
+    ```bash
+    helm repo add hashicorp https://helm.releases.hashicorp.com
+    helm install vault hashicorp/vault --version 0.9.1 --create-namespace -n m4d-system \
+        --set "server.dev.enabled=true" \
+        --values https://raw.githubusercontent.com/IBM/the-mesh-for-data/master/third_party/vault/plugin-secrets-kubernetes-reader/values.yaml \
+        --wait --timeout 120s
+    ```
+
+=== "OpenShift"
+
+    ```bash
+    helm repo add hashicorp https://helm.releases.hashicorp.com
+    helm install vault hashicorp/vault --version 0.9.1 --create-namespace -n m4d-system \
+        --set "global.openshift=true" \
+        --set "server.dev.enabled=true" \
+        --values https://raw.githubusercontent.com/IBM/the-mesh-for-data/master/third_party/vault/plugin-secrets-kubernetes-reader/values.yaml \
+        --wait --timeout 120s
+    ```
+
```

Comment:
What about the rest of the stuff for Vault like cluster role to access the k8s secrets?  will it be added in a follow-up PR?

Reply:
No, it will be here before this moves out of draft, I noticed this too and wanted to ask you if we can merge them into a single yaml file to apply? Or even add it to the m4d helm chart. Unsure.

Reply:
Yes, we can merge the yaml files. There is also the issue with creating the vault token secret and vault auth path configuration that is set in a script under hack/install/ and should be applied.

Reply:
We should work this out to be a simpler install before making a release.

Reply:
To summarize our discussion, for the upcoming release we will focus on blockers for the single cluster quick start and:
1. Add vault commands to a values file to be passed here
2. Add single YAML with all vault related resources
3. Add secret to the chart with login token to vault (parameter)
4. Remove secret provider